src/Project.hs

```diff
@@ -169,12 +171,17 @@ data PullRequestStatus
 newtype DeployEnvironment = DeployEnvironment Text
   deriving (Eq, Show, Generic)
 
+data DeploySubprojects
+  = AllSubprojects
```

Comment:
`AllSubproject` sounds a bit confusing to me if projects don't have subprojects, but only 1. What do you think about `EntireProject`? There is no confusion then for edge-cases.

Reply:
_Who am I to question a dutchman on the topic of bikesheds?_